### ISSUE TITLE
Support newer systems

### ISSUE THREAD

**Issue by klangtuch** [deadcast/UltraEd#38]
Right now UltraEd only works on old systems like Windows XP. This is because we rely on the N64 SDK. 
Although this is not an easy issue to fix, there are some possible ways to tackle this:
1. Find a way to run the N64 SDK on Windows 7+
1. Run a small virtual machine in the background that doesn't do anything except building the ROM

The second way should be easier and might work with ReactOS, we could investigate this is in the future.
Any thoughts on this @deadcast ? You probably know better than me what the issues in both approaches are.

**Comment by deadcast** [comment]
Yeah I would love to run some hidden virtual machine that is used just for the final ROM compilation stages! I tried applications like Wine and DosBox but those didn't support running the old 16-bit executables... :-;

Good idea about ReactOS. I should investigate that a little further but I'm afraid that path may still be doomed. Lol I have been very tempted to buy some industrial strength reverse engineering tools to just re-create the old, legacy tools. That way we could then possibly run on any OS! :) The dream I guess but in time perhaps.

**Comment by klangtuch** [comment]
As soon as the documentation is ready I'll play around with ReactOS. Maybe we have luck and it works :P

**Comment by deadcast** [comment]
@nterry Hello! Yes I do believe you could create a build of GCC with the appropriate flags. The problem I ran into was the final linker/rom building phase. That's where I ran into trouble since those are the secret, or were I guess thanks to @trhodeos great work, last tools needed to create the rom.

**Comment by trhodeos** [comment]
Yeah, still have a few issues to figure out, but take a look here: https://github.com/trhodeos/modern-n64sdk.. I opted for building tools in go for easy cross-platform building, but if it's a pain to get gcc set up on windows, it may be worth creating simple docker image w/ gcc+tools preinstalled.
